core/include/gnuradio-4.0/Block.hpp

```diff
@@ -206,7 +211,7 @@ concept BlockLike = requires(T t, std::size_t requested_work) {
     requires !std::is_copy_assignable_v<T>;
```

Comment:
Comment to the line 207: Is this `{ t.work(requested_work) } -> std::same_as<work::Result>;` obsolete now since you have `HasWork<t>` concept?

Reply:
removed